shopinvader_sale_cart/models/sale_order.py

```diff
@@ -37,7 +37,7 @@ def _find_open_cart(self, partner_id=None, uuid=None):
             # maybe a current cart exists with another uuid
             domain = self._get_open_cart_domain(partner_id, uuid=None)
             cart = self.search(domain, limit=1)
-        return cart
+        return cart.sudo()
```

Comment:
I prefer to avoid returning sudo'ed models (and sudo locally where it's needed) to avoid passing around sudoed models that can cause unexpected security holes by navigation through the relations.